Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:16.04
```

Comment:
Why not use python:3.6 or or [edxops/python:3.6](https://github.com/edx/edx-docker-base/blob/master/python/3.6/slim/Dockerfile)?